src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -9,5 +9,6 @@
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
+    public static final String MESSAGE_FOLDERS_LISTED_OVERVIEW = "%1$d folders listed!";
```

Comment:
Possible to make `folder` into singular or plural form based on the number of filtered folders returned?